cmd/arcstat/Makefile.am

```diff
@@ -8,6 +8,6 @@ dist_bin_SCRIPTS = arcstat
 #
 if USING_PYTHON_2
 install-exec-hook:
-	sed --in-place 's|^#!/usr/bin/env python3|#!/usr/bin/env python2|' \
+	sed ${ac_inplace} -e 's|^#!/usr/bin/env python3|#!/usr/bin/env python2|' \
 	    $(DESTDIR)$(bindir)/arcstat
```

Comment:
What is the fallback mechanism? This looks like, if `${ac_inplace}` is empty, it's just going to print the modified `arcstat` to stdout.

Reply:
`-i ''`

Reply:
Ah, yes - sorry, I mis-read the patch as `--in-place` vs nothing, rather than `--in-place` vs `-i`.